### ISSUE TITLE
Weekly goals

### ISSUE THREAD

**Issue by fpghost** [NerdyPepper/dijo#39]
If I have a habit I'm supposed to do 3 times per week instead of daily is it possible?

**Comment by NerdyPepper** [comment]
Yep, this is possible and it is on the list of things to do, over at #10. I will be tracking all feature requests over at that issue, so I will be closing this one if you don't mind.
